src/main/java/com/github/dariobalinzo/filter/Filter.java

```diff
@@ -0,0 +1,9 @@
+package com.github.dariobalinzo.filter;
+
+import java.util.Map;
+
+public interface Filter {
```

Comment:
Maybe DocumentFilter might be a better name for this interface. 
This might be a common interface for implementing https://github.com/DarioBalinzo/kafka-connect-elasticsearch-source/issues/10 in the future.

Reply:
Ok, will do it